### PR TITLE
Update LICENSE

### PR DESCRIPTION
**Pull request by FxKu**
Removing wrong author and update year

### CODE REVIEW

File: LICENSE [hunk 1]
```diff
@@ -18,4 +18,4 @@ FITNESS FOR A PARTICULAR PURPOSE AND NONINFRINGEMENT. IN NO EVENT SHALL THE
 AUTHORS OR COPYRIGHT HOLDERS BE LIABLE FOR ANY CLAIM, DAMAGES OR OTHER
 LIABILITY, WHETHER IN AN ACTION OF CONTRACT, TORT OR OTHERWISE, ARISING FROM,
 OUT OF OR IN CONNECTION WITH THE SOFTWARE OR THE USE OR OTHER DEALINGS IN THE
-SOFTWARE.
\ No newline at end of file
+SOFTWARE.
```
Comment (RafiaSabih) [inline LICENSE]: Isn't this a useless hunk?

Comment (FxKu) [reply LICENSE]: I used the online editor from GitHub #WasntMe

**Comment by Jan-M** [comment]
:+1:

**Comment by hjacobs** [comment]
:+1:

**Pull request merged**